src/pspm_convert_ecg2hb_amri.m

```diff
@@ -17,9 +17,9 @@
 %   ┌─────── options
 %   ├───────.channel: [optional, numeric/string, default as 'ecg']
 %   │                 Channel ID to be preprocessed.
-%   │                 Channel can be specified by its index in the given PsPM
-%   │                 data structure.
-%   │                 If there are multiple channels with 'ecg' type, only
+%   │                 Channel can be specified by its index (numeric) in the given
+%   │                 PsPM data structure or by its channel type (string).
```

Comment:
I adjusted the text here because it has said "specified by its index" in the beginning.